### ISSUE TITLE
cfn-lint fails to detect the unsupported \/ escape sequence 

### ISSUE THREAD

**Issue by trav-c** [aws-cloudformation/cfn-lint#1549]
*cfn-lint version: (`cfn-lint --version`)*
cfn-lint 0.32.1

*Description of issue.*
cfn-lint doesn't detect the use of \\/ (c-escape ns-esc-slash) inside double quotes which while legal in the YAML 1.2 spec and representing a literal slash, doesn't appear to be legal in YAML 1.0 or 1.1, or permitted by CloudFormation

The following fragment trips up the CloudFormation validation API, but passes on cfn-lint
```
Parameters:
  Test:
    Type: String
    Default: "Test escaped slash \/ string"
```

From what I can tell the use of \\/ inside double quotes is not permitted in property in the template (ie it's rejected in resource property in exactly the same way as in the Parameter defaults)

Using the native AWS CloudFormation ValidateTemplate API returns the error:
```
Template format error: YAML not well-formed. (line X, column Y)
```

ValidateTemplate permits \\/ in other contexts, such as single quotes where it doesn't have special meaning, and cfn-lint appears to reject other  illegal escape sequences (eg \\+) so perhaps it's just using a newer YAML spec version than CloudFormation for it's parsing and validation?

**Comment by kddejong** [comment]
@trav-c let me look into this and thanks for reporting this.  I'm guessing your initial assessment is correct.

**Comment by kddejong** [comment]
@trav-c let me look into this and thanks for reporting this.  I'm guessing your initial assessment is correct.  I've noticed this a few other times and I've been trying to keep them as sync as much as possible.
